browser-bunyan/package.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
You don't need this file. `package.json` is just for [special cases](https://github.com/DefinitelyTyped/DefinitelyTyped#i-notice-some-packages-having-a-packagejson-here).